docs/en/flight_modes/offboard.md

```diff
@@ -201,13 +281,13 @@ The following MAVLink messages and their particular fields and field values are
 ### Rover
 
 - [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED)
-  - The following input combinations are supported (in `type_mask`): <!-- https://github.com/PX4/PX4-Autopilot/blob/main/src/lib/FlightTasks/tasks/Offboard/FlightTaskOffboard.cpp#L166-L170 -->
+  - The following input combinations are supported (in `type_mask`):
```

Comment:
OK, I've touched this section because I believe you stated no mavlink API is supported. IF that is the case we should delete all of this and add a note either here (or more likely above at the top of the mavlink section) stating that offboard control via MAVLink is not supported. Right?

Reply:
PS Either way, that link I removed is broken.

Reply:
I decided we needed a note under the rover section for visibility :-). Better to be explicit.